content/docs/command-reference/init.md

```diff
@@ -14,106 +14,103 @@ DVC works best in a Git repository. This enables all features, providing the
-
-- Not enough isolation/granularity - commands like `dvc metrics diff`, `dvc dag`
-  and others by default dump all the metrics, all the pipelines, etc.
+- By default, DVC commands like `dvc checkout` and `dvc repro` explore the whole
+  <abbr>DVC repository</abbr> to find DVC-tracked data and pipelines to work
+  with. This can be undesirable and inefficient for large monorepos.
 
 #### How does it affect DVC commands?
 
-No matter what mode is used, DVC looks for the `.dvc` directory when it starts
-(from the current working directory and up). Location of the found `.dvc`
-directory determines the root of the DVC project. (In case of `--subdir` it
-might happen that Git repository root is located at different path than the DVC
-project root.)
+The <abbr>project</abbr> root is found by DVC by looking for `.dvc/` from the
+current working directory, up. It defines the scope of action for most DVC
+commands (e.g. `dvc repro`, `dvc pull`, `dvc metrics diff`), meaning that only
+`dvc.yaml`, `.dvc` files, etc. inside the project are usable by the commands.
 
-DVC project root defines the scope for most DVC commands. Mostly meaning that
-all `dvc.yaml` and `.dvc` files under the root path are being analyzed.
+With `--subdir`, the project root will be found before the Git root, making sure
+the scope of DVC commands run here is constrained to this project alone, even if
+there are more DVC-related files elsewhere in the repo. Similarly, DVC commands
+run outside this project root will ignore its contents.
 
-If there are multiple DVC sub-projects but they _are not_ nested, e.g.:
+If there are multiple `--subdir` projects, but not nested, e.g.:
 
-```
-.
+```dvc
+.           # plain Git repo
 ├── .git
-|
 ├── project-A
-│   └── .dvc
+│   ├── .dvc    # dvc init --subdir
 │   ...
 ├── project-B
-│   └── .dvc
+│   ├── .dvc    # dvc init --subdir
 │   ...
 ```
 
-DVC considers them a two separate DVC projects. Any DVC command that is being
-run in the `project-A` is not aware about DVC `project-B`. DVC does not consider
-Git repository root an initialized DVC project in this case and commands that
-require DVC project will raise an error.
-
-On the other hand, if there _are_ nested DVC projects, e.g.:
+DVC considers A and B separate projects. Any DVC command run in `project-A` is
+not aware of `project-B`. However, commands that involve versioning (like
+`dvc checkout`) access the commit history from the Git root (`.`).
 
-```
-project-A
-├── .dvc
-├── data-A.dvc
-│   ...
-└── project-B
-    ├── .dvc
-    ├── data-B.dvc
-    │   ...
-```
+> `.` is not a DVC project in this case, so most DVC commands can't be run
+> there.
 
-Nothing changes for the `project-B`. But for any DVC command being run in the
-`project-A` ignores the whole directory `project-B/`, meaning for example:
+If there are nested `--subdir`projects e.g.:
 
 ```dvc
-$ cd project-A
-$ dvc pull
+project-A       # full DVC + Git repo
```

Comment:
why do we need this comment?

Reply:
Hmmm they are informative. Changed to `git init && dvc init` in cac92e58. WDYT?

Reply:
so, that's probably what I don't understand - if we put them they highlight importance of something to my mind, I spend a cycle thinking why did author put a comment here - is .git important in this case? does it change anything?

Reply:
They do intent to highlight exactly what we mean by nested or not-nested --subdir projects:

Not nested: single-level DVC projects in a Git repo.
Nested: multi-level DVC projects inside (DVC projects inside) DVC/Git repo

This could be also just undetstood by looking at the directories shown (.git, .dvc) but it's not as obvious).

Reply:
.git still not important here to my mind unless I'm missing something. Nesting would work with `--no-scm` the same way effectively.

Reply:
I'm removing these block examples completely for now, and extracting to a following PR...